client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -526,29 +552,14 @@ export class PlanFeatures2023Grid extends Component {
 	}
 }
 
-PlanFeatures2023Grid.propTypes = {
-	isInSignup: PropTypes.bool,
-	onUpgradeClick: PropTypes.func,
-	// either you specify the plans prop or isPlaceholder prop
-	plans: PropTypes.array,
-	visiblePlans: PropTypes.array,
-	planProperties: PropTypes.array,
-	flowName: PropTypes.string,
-};
-
-PlanFeatures2023Grid.defaultProps = {
-	isInSignup: true,
-	onUpgradeClick: noop,
-};
-
 /* eslint-disable wpcalypso/redux-no-bound-selectors */
 export default connect(
-	( state, ownProps ) => {
+	( state, ownProps: any ) => {
 		const { placeholder, plans, isLandingPage, visiblePlans } = ownProps;
 
-		let planProperties = plans.map( ( plan ) => {
+		let planProperties: PlanProperties[] = plans.map( ( plan: any ) => {
```

Comment:
You don't need `any` here. In fact, you don't need to type `plan` at all since you know the type of `plans` (it's an array of strings). If you type `ownProps` above, this will be handled automatically. Otherwise you can type this better:

```suggestion
		let planProperties: PlanProperties[] = plans.map( ( plan: string ) => {
```